pkg/bwagreement/server.go

```diff
@@ -117,3 +128,21 @@ func (s *Server) GetBandwidthAllocations(ctx context.Context) (rows []*dbx.Bwagr
 	rows, err = s.DB.All_Bwagreement(ctx)
 	return rows, err
 }
+
+func (s *Server) verifySignature(ctx context.Context, ba *pb.RenterBandwidthAllocation) error {
```

Comment:
Do we also want to verify the payer bandwidth signature? @jtolds

Reply:
added the check